crates/ide-completion/src/completions/expr.rs

```diff
@@ -365,7 +365,8 @@ pub(crate) fn complete_expr_path(
                     add_keyword("false", "false");
 
                     if in_condition || in_block_expr {
-                        add_keyword("let", "let");
+                        add_keyword("let mut", "let mut $0");
```

Comment:
In postfix completion we call it `letm`, maybe better to do it here too for consistency.